EDMarketConnector.py

```diff
@@ -1867,7 +1867,11 @@ def show_killswitch_poppup(root=None):
         config.set('ui_transparency', 100)  # 100 is completely opaque
         config.delete('font')
```

Comment:
Looks like either config.py needs to handle "that key doesn't exist", or this needs a try/except:

```python
  File "C:\Users\Athan\Documents\Devel\EDMarketConnector\EDMarketConnector.py", line 1868, in <module>
    config.delete('font')
  File "C:\Users\Athan\Documents\Devel\EDMarketConnector\config.py", line 666, in delete
    winreg.DeleteValue(self.__reg_handle, key)
FileNotFoundError: [WinError 2] The system cannot find the file specified
```